gqueries/output_elements/output_series/table_space_requirements_transit_flows/space_requirements_transit_flows_exported_volume_oil_and_products.gql

```diff
@@ -0,0 +1,16 @@
+### Exported volume of oil and products (transit flow)
+# Returns total exported volume of oil and products in PJ. The difference between export and import makes up the transit flow of this group of carriers.
+
+- query = 
+    DIVIDE(
+        SUM(
+            V(energy_export_crude_oil_baseload, energy_export_crude_oil_backup, demand),
+            V(energy_export_diesel_baseload, energy_export_diesel_backup, demand),
+            V(energy_export_gasoline_baseload, energy_export_gasoline_backup, demand),
+            V(energy_export_heavy_fuel_oil_baseload, energy_export_heavy_fuel_oil_backup, demand),
+            V(energy_export_kerosene_baseload, energy_export_kerosene_backup, demand),
+            V(energy_export_lpg_baseload, energy_export_lpg_backup, demand),
```

Comment:
I see we query all oil products import flow here, even though, in the front-end, we label it only as 'diesel'. One of both should changed I think. What do you think @kaskranenburgQ / @kndehaan?

Reply:
Actually, I think all gqueries `space_requirements_transit_flows_exported_volume_xxx` and `space_requirements_transit_flows_imported_volume_xxx` are not even used somewhere... **Please check if this is correct.** @mabijkerk 

So only `space_requirements_transit_flows_space_requirement_xxx` are used for the space requirement tabel, and these directly query the specific node. So if this is correct, all the `space_requirements_transit_flows_exported_volume_xxx` and `space_requirements_transit_flows_imported_volume_xxx` queries can be deleted. 

To answer your question, the correct gquery `space_requirements_transit_flows_space_requirement_diesel` only encompasses the `land_use_per_unit` attribute of diesel. We also only received space requirement data (as well as cost data) of diesel, so I think we should only use diesel in both front-end as back-end. 

N.b.: for the costs aspects of transit flows, we actually used the label 'oil' or 'oil and products' rather than 'diesel' for the infrastructure costs in the detailled cost data chart which is in line with the structure of queries of energy carrier costs. However, only diesel is now assumed in the infrastructure costs of oil, but this is specified in the documentation.

Reply:
CORRECTION:
of the `space_requirements_transit_flows_imported_volume_xxx` queries, only the ones below are used in the table:
- `space_requirements_transit_flows_imported_volume_lng`
- `space_requirements_transit_flows_imported_volume_diesel`
- `space_requirements_transit_flows_imported_volume_liquid_hydrogen`
- `space_requirements_transit_flows_imported_volume_lohc`
- `space_requirements_transit_flows_imported_volume_ammonia`

all `space_requirements_transit_flows_exported_volume_xxx` are not used. 

**PLEASE CHECK IF CORRECT** @mabijkerk 

Conclusion remains the same, so assume diesel in front- and back-end (which is currently the case). All unused queries can thus be removed.

Reply:
Thanks for the explanation. Fine by me to remove the redundant queries.

Reply:
Redundant queries are now removed

Reply:
Redundant queries are now removed